contracts/spawn-caller/Cargo.toml

```diff
@@ -6,4 +6,4 @@ edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-ckb-std = { path = "../../" }
+ckb-std = { path = "../../", features=["build-with-clang", "dummy-atomic"] }
```

Comment:
Is "dummy-atomic" mandatory or optional?

Reply:
This is required. When using ckb-gen-types >= 0.117, the output will include the A instructions.

Reply:
When "dummy-atomic"  is set as default feature, bump major version (0.16).